.github/workflows/temp-ledger-emulator.yml

```diff
@@ -0,0 +1,30 @@
+name: Ledger Emulator Tests (temp)
+on:
+  push:
+    branches:
+      - AhaLabs:fix/troubleshooting-emulator-tests
+  pull_request:
+    branches:
+      - AhaLabs:fix/troubleshooting-emulator-tests
```

Comment:
Needs updating before merge.

Reply:
Just reverted the commit that adds this file